manifests/params.pp

```diff
@@ -9,8 +9,8 @@
   # should foreman manage host provisioning as well
   $unattended     = true
   $unattended_url = undef
-  # Enable users authentication (default user:admin pw:changeme)
-  $authentication = true
+  # Enable users authentication (default user:admin pw:changeme) (Deprecated)
```

Comment:
I'd prefer to mention it's unused starting 1.21 so we know when to remove it.

Reply:
updated